### DISCUSSION TITLE
how add add New Fluid

### DISCUSSION THREAD

**Discussion by Nouman090** [oemof/tespy#349] [Q&A]
Is there any way to add new fluid or specify any fluid property like specific enthalpy?

**Comment by fwitte** [comment]
Dear @Nouman090,

currently there is no easy way to do this. There are some plans for the future however. If you are interested, you could join the next user meeting in August, I‘ll prepare some information on how the software is built and where and what changes need to be implemented.

Best

Francesco

**Reply by theo-bech** [reply]
Hi, have there been any updates on that? Or, if not, what would be the non-easy way to do that?

Best,
Theo

**Reply by fwitte** [reply]
Hi Theo, unfortunately not really. Except those information I wrote under #407. Maybe also interesting for you @Nouman090.
